src/js/components/List/List.js

```diff
@@ -245,7 +243,15 @@ const List = React.forwardRef(
             as={as || 'ul'}
             itemFocus={itemFocus}
             tabIndex={onClickItem || onOrder ? 0 : undefined}
-            onBlur={onOrder ? () => setActive(undefined) : undefined}
+            onFocus={() =>
+              !active && active !== 0
```

Comment:
So, if `active === undefined`, we are going to `setActive(active)`? That seems odd. Can we not just do `onFocus={() => setActive(lastActive)}`? Why do we need to check `active`?

Reply:
After implementing `onFocus={() => setActive(lastActive)}` the issue that would arise was:

1. You selected / clicked on an item
2. You tab out of the table / click out of the table
3. You hold down a click on a different item and the active styling would show up on the previous item (in this case `lastActive`)

And one of the issues that we realized on the most recent deployment preview was the fact that clicking the first entry (or selecting it via keyboard) would not retain its active styling after closing the modal as the code would read the `0th` index as `undefined`, thus messing up the entire chain of `onFocuses` and `onBlurs`. After logging out the exact orders and values of what `active` and `lastActive` were, doing the check here eliminates `undefined` getting passed down the chain of functions. While it looks out of place, when logging out the chain of events that occurs when a user clicks, closes the modal, and clicks / tabs out of the table, the check is necessary to pass the correct indices of what is active or what was lastActive.

Reply:
OK. Can you add a brief comment to the code then? This could be helpful detail down the road.

Reply:
No problem, will do!